hsctl

```diff
@@ -94,20 +92,20 @@ solr_schema_hs() {
     echo
     echo "*** Setting up solr schema ***"
     echo "*** From hydroshare: build_solr_schema ***"
-    docker exec hydroshare_hydroshare_1 python manage.py build_solr_schema -f schema.xml
+    docker exec hydroshare python manage.py build_solr_schema -f schema.xml
     sleep 1s
     echo "*** From solr: mv schema.xml to /opt/solr/example/solr/collection1/conf/ ***"
-    docker exec hydroshare_solr_1 cp /hydroshare/schema.xml /opt/solr/example/solr/collection1/conf/schema.xml
+    docker exec solr cp /hydroshare/schema.xml /opt/solr/example/solr/collection1/conf/schema.xml
 }
 
 start_hs() {
     echo "*** ${1^^}: all docker containers ***"
     preflight_hs
     for f in "${OTHER_DOCKER_CONTAINERS[@]}"; do
-        docker start $f;
+        docker-compose start $f;
```

Comment:
@selimnairb - we had an issue with `docker-compose up -d` call as it tended to fully rebuild the container and not simply bring it up again. The `start` call is equivalent whether invoked via docker-compose or docker since it is bringing up an already built and defined container.